### PR TITLE
Move -dev readme info into the main readme, update with required brew…

### PR DESCRIPTION
**Pull request by ilyavw**
… install

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -3,6 +3,10 @@ version: '3'
 services:
   db:
     image: postgres:alpine
+    ports:
+      - "5432:5432"
+    volumes:
+      - /tmp/postgresql:/var/lib/postgresql
```
Comment (infinitewarp) [inline docker-compose.yml]: For the local side, `${TMPDIR-/tmp}` is safer instead of the absolute `/tmp`.

Comment (infinitewarp) [reply docker-compose.yml]: Actually… I'm not sure how that translates into the yaml world, but the suggestion is still sound. ¯\\\_(ツ)\_/¯

Comment (ilyavw) [reply docker-compose.yml]: Docker won't use it by default, it requires extra configuration. 

https://docs.docker.com/docker-for-mac/osxfs/#namespaces

**Comment by infinitewarp** [comment]
A nice demo of this would include something like:

```
make clean
make reinitdb

ARN_WITH_RUNNING="arn:aws:iam::518028203513:role/grant_cloudi_to_372779871274"
./cloudigrade/manage.py add_account ${ARN_WITH_RUNNING}
```

**Pull request merged**